staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -254,14 +254,8 @@ type ObjectMeta struct {
 	// +patchStrategy=merge
 	Finalizers []string `json:"finalizers,omitempty" patchStrategy:"merge" protobuf:"bytes,14,rep,name=finalizers"`
 
-	// Deprecated: ClusterName is a legacy field that was always cleared by
-	// the system and never used; it will be removed completely in 1.25.
-	//
-	// The name in the go struct is changed to help clients detect
-	// accidental use.
-	//
-	// +optional
-	ZZZ_DeprecatedClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
+	// Tombstone: ClusterName was a legacy field that was always cleared by
+	// the system and never used. It consumed proto tag 15.
```

Comment:
Include the json field name as well. In other places, we actually kept the full field declaration, just commented out... dunno if we want to do that here.

```
	// IPFamily is tombstoned to show why 15 is a reserved protobuf tag.
	// IPFamily *IPFamily `json:"ipFamily,omitempty" protobuf:"bytes,15,opt,name=ipFamily,Configcasttype=IPFamily"`
```